spec/requests/categories_controller_spec.rb

```diff
@@ -444,6 +444,36 @@
           category.reload
           expect(category.required_tag_group).to be_nil
         end
+
+        it "does not update other fields" do
+          SiteSetting.tagging_enabled = true
+          tag_group_1 = Fabricate(:tag_group)
+          tag_group_2 = Fabricate(:tag_group)
+
+          category.update!(
+            allowed_tags: ["hello", "world"],
+            allowed_tag_groups: [tag_group_1.name],
+            required_tag_group_name: tag_group_2.name
+          )
+
+          put "/categories/#{category.id}.json"
```

Comment:
```suggestion
          put "/categories/#{category.id}.json"

          expect(response.status).to eq(200)
```

Testing for the response helds to provide better feedback in the event that a failure occurs.